susemanager-frontend/susemanager-nodejs-sdk-devel/susemanager-nodejs-sdk-devel.changes

```diff
@@ -52,12 +53,12 @@ Wed Aug 30 15:40:14 UTC 2017 - mc@suse.com
 -------------------------------------------------------------------
 Wed Aug  9 12:32:36 UTC 2017 - dmaiocchi@suse.com
 
-- FIX: remove the 2 time contained "gulp-concat": "2.6.0" 
```

Comment:
It doesn't matter and it does not hurt in this case, but please take care what your IDE is doing here. I don't think we have such a checker for changes of existing changes in the the changelog (sorry for the pun), but we should not change whatever is already in there.